cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -22,20 +22,20 @@ package app
 import (
 	"context"
 	"fmt"
-	"k8s.io/apimachinery/pkg/runtime/schema"
 	"math/rand"
 	"net/http"
 	"os"
 	"sort"
 	"time"
 
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
```

Comment:
this is a strange order. Should be below in the "kube block".

Reply:
apologies my plugin for ordering imports bugged out, fixed.